modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -25,6 +25,7 @@ public class {{classname}} {{#parent}}extends {{{.}}}{{/parent}}{{^parent}}{{#ha
   {{/jackson}}
   {{#gson}}
   @SerializedName("{{baseName}}")
+  @SerializedName("{{baseName}}")
```

Comment:
@carnevalegiacomo is this line added by mistake?

Reply:
yes

Reply:
can you please remove it when you've time?

Reply:
https://github.com/OpenAPITools/openapi-generator/pull/11162/commits/5813fb18440c2c062752075b39bb3f710d8c07d0